src/index.tsx

```diff
@@ -209,6 +212,7 @@ const ReactInputVerificationCode = ({
               i === activeIndex ? 'is-active' : ''
             }`}
             onFocus={onItemFocus(i)}
+            data-cy={`${dataCy}-item`}
```

Comment:
wdyt about adding the index?

```suggestion
            data-cy={`${dataCy}-${i}-item`}
```

Reply:
Great idea 🚀